python/src/wslink/launcher.py

```diff
@@ -249,10 +249,10 @@ def filterResponse(obj, public_keys):
 # -----------------------------------------------------------------------------
 
 def extractSessionId(request):
-    path = request.path.split('/')
+    path = request.path.split(b'/')
     if len(path) < 3:
-       return None
-    return str(path[2])
+        return None
+    return str(path[2],'utf-8')
```

Comment:
This is not valid in python 2.7. Using `.encode('utf-8')` works, but `str(path[2])` will give you the wrong thing in python3.

I suggest converting up top, `request.path.decode('utf-8')` if type is bytes